# Message header
Header header

float64[] distance
float64[] direction
float64[] risk_value
